### DISCUSSION TITLE
Can we highlight fields ?

### DISCUSSION THREAD

**Discussion by schavalileaf** [wojtekmaj/react-pdf#1799] [Q&A]
Can we highlight pdf fields when found by name in code

**Comment by wojtekmaj** [comment]
Can you give an example of what you're trying to achieve?

**Reply by schavalileaf** [reply]
I have set of dropdowns each representing a field (or annotation) of a pdf.
When I select the dropdown it should automatically highlight the corresponding field on pdf.

Text highlighting is possible, but I want field/annotation highlighting.


I tried using document.queryselector(.react-pdf__page_annotations)
it works when i read through all children of annotations and highlight when I find the name I want but only when renderform =true.
#1. Requirement is to highlight without being in editable mode.
#2. This way the selector only gives annotations of 1 page or current page only, we need to highlight in all pages.
